v2/internal/controllers/network_test.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	network "github.com/Azure/azure-service-operator/v2/api/network/v1api20201101"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1api20200601"
+	"github.com/Azure/azure-service-operator/v2/internal/genericarmclient"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func newLoadBalancerForVMSS(
```

Comment:
Is this only exposing helper methods now? Why not move this? Who is consuming these?

Reply:
After moving the majority of the tests, I discovered some cross-wiring where other tests were mis-reusing helper methods that should have been private to other tests. I brought back into this package just the required helpers.